drivers/watchdog/wdt_mcux_imx_wdog.c

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (C) 2020, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nxp_imx_wdog
+
+#include <drivers/watchdog.h>
+#include <fsl_wdog.h>
+
+#define LOG_LEVEL CONFIG_WDT_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(wdt_mcux_wdog);
+
+#define WDOG_TMOUT_SEC(x)  ((int)(x * 2 / MSEC_PER_SEC) - 1)
```

Comment:
The cast to `int` is not necessary.

Reply:
The cleanup is ok, but the test still doesn't pass on mimxrt1050_evk

Reply:
Updated with no cast to int